### PR TITLE
PKGBUILD: desurium: we cannot depend on package groups. Sorry for the br...

### PR DESCRIPTION
**Pull request by matthiaskrgr**
...ainfart.

### CODE REVIEW

**Comment by matthiaskrgr** [comment]
Hmm don't merge yet I'm going to do a bit more stuff.

**Comment by matthiaskrgr** [comment]
ok should be ready now

**Comment by karolherbst** [comment]
is there an arch bug for this issue?

File: distro/archlinux/git/PKGBUILD [hunk 1]
```diff
@@ -22,7 +22,7 @@ _gitname=Desurium
 
 
 if [ "${CARCH}" = "x86_64" ]; then
-    depends+=('binutils-multilib' 'multilib-devel')
+	depends+=('binutils-multilib' 'gcc-multilib' 'libtool-multilib')
```
Comment (karolherbst) [inline distro/archlinux/git/PKGBUILD:25]: is here a whitespace error?

**Comment by karolherbst** [comment]
PKGBUILD is now maintained in AUR

**Pull request closed without merging**